https://issues.redhat.com/browse/ACM-4933 Updating certs doc (#5221)
open-cluster-management/rhacm-docs: governance/certificates.adoc

[#certificates]
= Certificates

> All certificates required by services that run on {product-title-short} are created during the installation of {product-title-short}. You must create a Kubernetes TLS Secret for your certificate. After you create your certificates, you can replace certain certificates that are created by the {product-title-short} installer.Certificates are created and managed by the following components:

Reviewer: ```suggestion
All certificates required by services that run on {product-title-short} are created during the installation of {product-title-short}. Certificates are created and managed by the following components of the OpenShift platform:
```

* OpenShift Service Serving Certificates
* {product-title-short} webhook controllers
* Kubernetes Certificates API
* OpenShift default ingress

*Required access*: Cluster administrator

Continue reading to learn more about certificate management:
 
- <<acm-certs,{product-title-short} hub cluster certificates>>
- <<acm-component-cert,{product-title-short} component certificates>>
- <<acm-managed-cert,{product-title-short} managed certificates>>

*Note:* Users are responsible for certificate rotations and updates.

[#acm-certs]
== {product-title-short} hub cluster certificates

OpenShift default ingress certificate is considered a hub cluster certificate. After {product-title-short} is installed, observability certificates are created and used by the observability components to provide mutual TLS on the traffic between the hub cluster and managed cluster. The Kubernetes secrets that are associated with the observability certificates. 

> * The `open-cluster-management-observability` namespace contain the following certificates:

Reviewer 2: ```suggestion
* The `open-cluster-management-observability` namespace contains the following certificates:
```

* Non-CA certificates are renewed automatically when the remaining valid time is no more than 73 days. After the certificate is renewed, the pods in the related deployments restart automatically to use the renewed certificates.

* CA certificates are renewed automatically when the remaining valid time is no more than one year. After the certificate is renewed, the old CA is not deleted but co-exist with the renewed ones. Both old and renewed certificates are used by related deployments, and continue to work. The old CA certificates are deleted when they expire.

* When a certificate is renewed, the traffic between the hub cluster and managed cluster is not interrupted.

[#acm-component-cert]
== {product-title-short} component certificates

View the following table for a summarized list of the component pods that contain certificates and whether a secret needs to be deleted prior to restarting the pod:

.Pods that contain {ocp-short} managed certificates
|===
| Service name | Namespace | Sample pod name | Secret name (if applicable)

> | channels-apps-open-cluster-management-webhook-svc

Author: @gparvin I believe this is the table you were referring to

The managed cluster certificates are refreshed automatically.

[#additional-resources-cert]
== Additional resources

- See xref:../governance/cert_manage_overview.adoc#cert-manage-overview[Managing certificates overview] for topics.

- Use the certificate policy controller to create and manage certificate policies on managed clusters. See xref:../governance/cert_policy_ctrl.adoc#certificate-policy-controller[Certificate policy controller] for more details.

//this link will take the user to a different book, after reading the contents in the section, this link seems appropriate
- See link:../applications/configuring_git_channel.adoc#using-custom-CA-certificates-for-secure-HTTPS-connection[Using custom CA certificates for a secure HTTPS connection] for more details about securely connecting to a privately-hosted Git server with SSL certificates.

//not sure about linking to the troubleshoot topic; might delete this reference

> - Refer to link:../troubleshooting/trouble_cluster_offline_cert.adoc#troubleshooting-imported-clusters-offline-after-certificate-change[Troubleshooting imported clusters offline after certificate change] topic for more details.  

Author: Is this so common that we need to add this to the procedure doc? asked by @swopebe during live review
